app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Good job so far. However, I'm still unable to confirm my email after signing up. Kindly implement the email confirmation such that the email confirmation message is sent to the terminal with the link to confirm the email. Users can click on the link to confirm their email from the terminal, and it takes them to the browser. Kindly test that it works before requesting another review.

Reply:
@julie-ify , thanks for your review. My feature works but by sending an email to mailtrap.io. I will implement your new requirement so it prints it to the terminal.

Reply:
@julie-ify, I've recorded a loom [video](https://www.loom.com/share/d149da88fcb54216ad79d292aded1eca),  the URL and token are displaying in the terminal.